7.9.вложенные_циклы_часть_2/7.9.2.py

```diff
@@ -0,0 +1,10 @@
+num = int(input())
+for i in range(1, num + 1):   
+    counter = 0               
+    for j in range(i):        
+        counter += 1          
+        print(counter, end="")
+    for k in range(i, 1, -1): 
+        counter -= 1          
+        print(counter, end="")
```

Comment:
`j` и `k` не используешь:
```suggestion
    for _ in range(i):        
        counter += 1          
        print(counter, end="")
    for _ in range(i, 1, -1): 
        counter -= 1          
        print(counter, end="")
```